Sources/JWTKeychain/Support/UserAuthenticating.swift

```diff
@@ -50,7 +56,7 @@ public class UserAuthenticator: UserAuthenticating {
     /// - Returns: the updated user
     public func update(request: Request, hasher: HashProtocol) throws -> U {
```

Comment:
I'm ok with changing to camelCase but isn't it more common in JSON world to use snake case?